### ISSUE TITLE
Result and code problems

### ISSUE THREAD

**Issue by Tao-Yang7** [HotanLee/DeFT#2]
Thank you for your great work！
While replicating your code for the Tiny-ImageNet dataset, I observed that after Phase 1 training, the model's precision and recall performance appeared normal. However, during Phase 2 training, the training loss was minimal right from the start. Additionally, when sym=0.2, the model's final accuracy was only around 0.5. I would like to understand the possible reasons for this issue is.   2: I noticed that in dataloader_cub.py and dataloader_stanford_cars.py, your code does not seem to handle the case when noisy_mode=idn, which causes the program to crash. I would like to know how to reproduce the results in your paper for these two datasets when noisy_mode=idn. Looking forward to your reply!

**Comment by HotanLee** [comment]
Thanks for your attention!
1. Are the results on the other dataset normal? The Phase 2 just FFT the pre-trained model on selected samples. Since the results of Phase 1 are normal, there should be no problem in the second phase. Please check if the config is correct.
2. Sorry for the negligence. I used an additional file to generate the idn noise for these two datasets and forgot to upload it. I have updated the dataloader files. Please check if the codes work properly.

**Comment by Tao-Yang7** [comment]
Thans for your reply!

1. I found that I encountered some issues with dataset loading, which led to this result. It is now working properly. Thank you for your response.
2.Thanks, it works properly.



I encountered a new issue. I am using a 24GB NVIDIA GeForce RTX 3090 GPU, and during the training process of the clothing1m dataset, the phase 1 training log is as follows:
Epoch [  1/ 10] Iter[1000/1001]  total-loss: 3.49276
| Test Epoch #1  Accuracy: 70.60

Epoch [  2/ 10] Iter[1000/1001]  total-loss: 3.3057
| Test Epoch #2  Accuracy: 71.39

Epoch [  3/ 10] Iter[1000/1001]  total-loss: 3.0550
| Test Epoch #3  Accuracy: 70.31

Epoch [  4/ 10] Iter[1000/1001]  total-loss: 2.9331
| Test Epoch #4  Accuracy: 71.13

Epoch [  5/ 10] Iter[1000/1001]  total-loss: 3.0961
| Test Epoch #5  Accuracy: 70.47

Epoch [  6/ 10] Iter[1000/1001]  total-loss: 0.3081
| Test Epoch #6  Accuracy: 71.68

Epoch [  7/ 10] Iter[1000/1001]  total-loss: 0.0964
| Test Epoch #7  Accuracy: 69.24

Epoch [  8/ 10] Iter[1000/1001]  total-loss: 0.1510
| Test Epoch #8  Accuracy: 71.35

Epoch [  9/ 10] Iter[1000/1001]  total-loss: 0.2705
| Test Epoch #9  Accuracy: 69.15

Epoch [ 10/ 10] Iter[1000/1001]  total-loss: 0.0989
| Test Epoch #10         Accuracy: 70.85


The phase 2 training log is as follows:
Loading CLIP (backbone: ViT-B/16)
Total params: 124350991
Image Tuned params: 85065230
Tuned params (w/o head): 85054464

Epoch [  1/  5] Iter[1000/1001]  total-loss: 0.5488
| Test Epoch #1  Accuracy: 69.36

Epoch [  2/  5] Iter[1000/1001]  total-loss: 0.1974
| Test Epoch #2  Accuracy: 69.40

Epoch [  3/  5] Iter[1000/1001]  total-loss: 0.0193
| Test Epoch #3  Accuracy: 69.41

Epoch [  4/  5] Iter[1000/1001]  total-loss: 0.1093
| Test Epoch #4  Accuracy: 70.27

Epoch [  5/  5] Iter[1000/1001]  total-loss: 0.0389
| Test Epoch #5  Accuracy: 68.32

Best Acc: 70.27 Last Acc: 68.32
I noticed that the accuracy is lower than the 72.44 mentioned in your paper, and phase 1 accuracy is higher than phase 2 accuracy. I am sure my configuration file is correct, and I would like to know the possible reasons for these results. I look forward to your reply.

**Comment by HotanLee** [comment]
Sorry again for the mistake. The _batch_size_ in the config is wrong, please set it to 64. 
Besides, in some settings, the ACC of PEFT is similar to or even higher than that of FFT, e.g., _sym0.2_ on Tiny-ImageNet (See Figure 3(b) for reference).

**Comment by Tao-Yang7** [comment]
Thank you for your response. I'm curious about the possible reasons why the ACC of PEFT is higher than that of FFT. If PEFT can filter out a clean dataset, why does the performance after FFT on a clean dataset turn out to be worse? Could this be related to the dataset?

**Comment by HotanLee** [comment]
I think the main reason is the data size. In the FFT phase, we do not make use of the noisy samples, and the FFT always needs more samples for optimization. This may lead to a degraded performance and could be improved with semi-supervised methods.
